modules/webhooks/app/workers/represented_webhook_job.rb

```diff
@@ -66,16 +69,14 @@ def perform(webhook_id, resource, event_name)
       url: webhook.url,
       request_headers: headers,
       request_body: body,
-      response_code: response.try(:code).to_i,
-      response_headers: response.try(:headers),
-      response_body: response.try(:to_s) || exception.try(:message)
+      response_code: response&.status,
+      response_headers: response&.headers&.to_h&.transform_keys {|k| k.underscore.to_sym},
```

Comment:
rubocop -a modules/webhooks/app/workers/represented_webhook_job.rb 😄 
```suggestion
      response_headers: response&.headers&.to_h&.transform_keys { |k| k.underscore.to_sym },
```